front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -59,83 +51,19 @@ const CustomFieldTitle = styled.div`
   font-weight: 500;
 `;
 
-const ChevronIcon = styled(Icon)`
-  width: 12px;
-  height: 12px;
-  fill: ${colors.label};
-  margin-left: 20px;
-  transition: fill 80ms ease-out, transform 200ms ease-out;
+const StyledAccordion = styled(Accordion)`
```

Comment:
This should not be needed - borders are defined in the accordion component and should act as expected.

Reply:
We seem to have double borders somewhere. I can look into which other component might be bringing them.

Reply:
The problem is the container on top - accordions should not be wrapped in a container to work properly. Let's delete it?